ci/cpu/build.sh

```diff
@@ -5,19 +5,20 @@
 ######################################
 set -e
 
-# Logger function for build status output
-function logger() {
-  echo -e "\n>>>> $@\n"
-}
-
 # Set path and build parallel level
-export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
-export PARALLEL_LEVEL=4
-export CUDF_HOME="${WORKSPACE}/cudf"
```

Comment:
was `export CUDF_HOME="${WORKSPACE}/cudf"` intentionally removed here?

Reply:
it wasn't intentional.  line was replaced.